chia/util/initial-config.yaml

```diff
@@ -287,6 +287,10 @@ full_node:
   # timeout for weight proof request
   weight_proof_timeout: 360
 
+  # List of trusted DNS seeders to bootstrap from.
+  # If you modify this, please change the hardcode as well from FullNode.set_server()
+  dns_servers:
+    - "seeder.fchirica.com"
```

Comment:
we should update this to `seeder.chia.net` before we land this, I think.

Reply:
We actually want a list of approved dns servers that will include a chia.net one and eg a Hoffmang.com one etc. I have a couple of volunteers lined up